tools/cell_census_builder/tests/conftest.py

```diff
@@ -104,13 +107,13 @@ def soma_path(tmp_path: pathlib.Path) -> str:
 def datasets(assets_path: str) -> List[Dataset]:
     datasets = []
     for organism in ORGANISMS:
-        for dataset_id in range(2):
-            h5ad = get_h5ad(organism)
-            h5ad_path = f"{assets_path}/{organism.name}_{dataset_id}.h5ad"
+        for i in range(2):
```

Comment:
add `NUM_DATASETS` constant and
```suggestion
        for i in range(NUM_DATASETS):
```